app/gateway/traffic-control/proxy.md

```diff
@@ -0,0 +1,181 @@
+- `read_timeout`: Defines, in milliseconds, a timeout between two
+  successive read operations for receiving a request from your upstream
+  service.  Defaults to `60000`.
+
+{{site.base_gateway}} will send the request over HTTP/1.1 and set the following headers:
+
+<!--vale off-->
+
+| Header                                  | Description |
+|-----------------------------------------|-------------|
+| `Host: <your_upstream_host>`            | The host of your Upstream. |
+| `Connection: keep-alive`                | Allows for reusing the Upstream connections. |
+| `X-Real-IP: <remote_addr>`              | Where `$remote_addr` is the variable bearing the same name provided by [ngx_http_core_module](https://nginx.org/docs/http/ngx_http_core_module.html#var_remote_addr). The `$remote_addr` is likely overridden by [ngx_http_realip_module](https://nginx.org/docs/http/ngx_http_realip_module.html). |
+| `X-Forwarded-For: <address>`            | `<address>` is the content of `$realip_remote_addr` provided by [ngx_http_realip_module](https://nginx.org/docs/http/ngx_http_realip_module.html) appended to the request header with the same name. |
+| `X-Forwarded-Proto: <protocol>`         | `<protocol>` is the protocol used by the client. If `$realip_remote_addr` is one of the **trusted** addresses, the request header with the same name gets forwarded if provided. Otherwise, the value of the `$scheme` variable provided by [ngx_http_core_module](https://nginx.org/docs/http/ngx_http_core_module.html#var_scheme) will be used. |
+| `X-Forwarded-Host: <host>`              | `<host>` is the host name sent by the client. If `$realip_remote_addr` is one of the **trusted** addresses, the request header with the same name gets forwarded if provided. Otherwise, the value of the `$host` variable provided by [ngx_http_core_module](https://nginx.org/docs/http/ngx_http_core_module.html#var_host) will be used. |
+| `X-Forwarded-Port: <port>`              | `<port>` is the port of the server which accepted a request. If `$realip_remote_addr` is one of the **trusted** addresses, the request header with the same name gets forwarded if provided. Otherwise, the value of the `$server_port` variable provided by [ngx_http_core_module](https://nginx.org/docs/http/ngx_http_core_module.html#var_server_port) will be used. |
+| `X-Forwarded-Prefix: <path>`            | `<path>` is the path of the request which was accepted by {{site.base_gateway}}. If `$realip_remote_addr` is one of the **trusted** addresses, the request header with the same name gets forwarded if provided. Otherwise, the value of the `$request_uri` variable (with the query string stripped) provided by [ngx_http_core_module](https://nginx.org/docs/http/ngx_http_core_module.html#var_server_port) will be used. **Note**: {{site.base_gateway}} returns `"/"` for an empty path, but it doesn't do any other normalization on the request path. |
+| All other headers | Forwarded as-is by {{site.base_gateway}} | 
+<!--vale on-->
```

Comment:
```suggestion

<!--vale on-->
```

@cloudjumpercat for some reason adding a new line here fixes the rendering issue 🤷